Documentation/xcassets/swift2.md

```diff
@@ -21,6 +21,8 @@ You can customize some elements of this template by overriding the following par
 | Parameter Name | Default Value | Description |
 | -------------- | ------------- | ----------- |
 | `enumName` | `Asset` | Allows you to change the name of the generated `enum` containing all assets. |
+| `colorTypeName` | `ColorAsset` | Allows you to change the name of the generated color container type. |
```

Comment:
"of the struct type representing a color"?

_(I find the current wording a bit confusing wrt the difference between `enumName` and `[color|image]TypeName` otherwise)_